Changes

```diff
@@ -178,6 +178,10 @@ Working version
   Updates usage of sigprocmask to pthread_sigmask in otherlibs/unix.
   (Max Slater, review by Miod Vallat and Xavier Leroy)
 
+- #?????: Unify MSVC and MinGW-w64 code paths, by always using WinAPI
+  directly.
+  (David Allsopp, Antonin Décimo, and Samuel Hym, review by ????)
```

Comment:
The wording of this changelog entry seems to imply that no other places remain in the code with separate MinGW/MSVC codepaths? Is that the case? If not, I would suggest being more precise (eg mention that this is just about `usleep`/`Sleep` or write "Unify _some_ MSVC and MinGW-w64 code paths, ...".

Reply:
Well, we were quite surprised that it seems to be the _only_ place in the runtime where there were/could have been different code paths involving `<unistd.h>`.